src/qt/overviewpage.cpp

```diff
@@ -124,8 +124,8 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
 {
     ui->setupUi(this);
 
-    // use a SingleColorIcon for the "out of sync warning" icon
-    QIcon icon = platformStyle->SingleColorIcon(":/icons/warning");
+    // use red icon for the "out of sync warning" icon
+    QIcon icon(":/icons/outofsync");
```

Comment:
No, SVG doesn't work without extra work on our part. Qt doesn't handle it natively.

But I think you're right about the ColorizeIcon thing, so I did that. I added it as a static method on PlatformStyle instead of making it a global function. Not that forced, unnatural OOP is good, but it seems better than putting the global namespace.